mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -183,4 +183,7 @@ SearchException cannotFindReadableProperty(@FormatWith(PojoTypeModelFormatter.cl
 	@Message(id = ID_OFFSET_2 + 15,
 			value = "Invalid property handle factory name: '%1$s'. Valid names are: %2$s.")
 	SearchException invalidPropertyHandleFactoryName(String invalidRepresentation, List<String> validRepresentations);
+
+	@Message(id = ID_OFFSET_2 + 16, value = "Error trying accessing Hibernate ORM session." )
```

Comment:
I really think it's "Error trying to access" rather than "Error trying accessing".

"trying accessing" would mean "accessing" is something we wanted to try for the experience it gives us, because it's fun, something we've never done, etc. Which doesn't make much sense here.
 
See https://learnenglish.britishcouncil.org/intermediate-grammar/verbs-followed-ing-or-infinitive-2